lua/roslyn/init.lua

```diff
@@ -43,7 +43,29 @@ function M.setup(config)
                 local root = utils.root(opt.buf)
                 vim.b.roslyn_root = root
 
-                local solution = utils.predict_sln_file(root)
+                local multiple, solution = utils.predict_target(root)
+
+                if multiple then
+                    -- If the user has `lock_target = true` then wait for them
+                    -- to choose a target explicitly before starting the LSP.
+                    --
+                    -- For `lock_target = false`, being asked to choose a target
+                    -- on every opened file would be annoying, so fall back to
+                    -- default handling.
+                    if roslyn_config.lock_target then
+                        vim.notify(
```

Comment:
I think we can just notify with this no matter what before the if statement, and then just have
```lua
if roslyn_config.lock_target then
    return
end
```

Reply:
We could, but I changed the messages to be slightly different. When `lock_target`, the message is

> Multiple potential target files found. Use `:Roslyn target` to select a target.

because we don't select anything. Whereas for `not lock_target` the message is

> Multiple potential target files found. Use `:Roslyn target` to change the target for the current buffer.

which asks users to _change_ target rather than _select_ one, because we will have launched the LSP on the fallback path (i.e, nearest project or currently open target).

Reply:
Okay, fair enough🙌